### DISCUSSION TITLE
Jogging with g0 rather than g1?

### DISCUSSION THREAD

**Discussion by Trakyan** [winder/Universal-G-Code-Sender#2228] [Help]
I've noticed UGS jogs with g1 commands rather than g0, which is a bit of a problem for some setups since g1 is a cutting command and activates the tool based on the s parameter. My googling seems to indicate it's possible to switch between the two but I can't find how exactly.
Thanks for any help.

**Comment by breiler** [comment]
Sorry, I thought I answered your forum post here https://groups.google.com/g/universal-gcode-sender/c/q5y17aoPKuM/m/fAJREY8qAgAJ but it must have vanished into the void...

The post that you found (https://github.com/sir-buckyball/chrome-gcode-sender/issues/30) is for "chrome-gcode-sender" and not UGS. There is no way to switch between the two commands in UGS.

We are using G1 for jogging as this is the only way to have adjustable speeds. With G0 it will always be full throttle. I don't think that we will change this anytime soon as this will effect large portions of UGS.

As another option, would it work to use the configuration `ON_BOOT.GCODE` in smoothieware (http://smoothieware.org/on_boot.gcode) to send `M5` on startup?
